Tests/Unit/DSL/ParametersTraitTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\Unit\DSL;
+
+use ONGR\ElasticsearchBundle\DSL\ParametersTrait;
+
+/**
+ * Test for ParametersTrait.
+ */
+class ParametersTraitTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var ParametersTrait
+     */
+    private $mock;
```

Comment:
Name is incorrect. Use more defining names. `parametersTraitMock`should be better.